app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_command = command.split("")
```

Comment:
There is no need to pass an argument here, since you should split by space and it is a default option